pom.xml

```diff
@@ -162,51 +127,37 @@
         </dependency>
 
         <dependency>
-            <groupId>org.flywaydb</groupId>
-            <artifactId>flyway-core</artifactId>
-            <version>3.2.1</version>
-        </dependency>
-
-        <dependency>
             <groupId>org.hibernate</groupId>
-            <artifactId>hibernate-entitymanager</artifactId>
-            <version>4.3.10.Final</version>
-        </dependency>
-
-        <dependency>
-            <groupId>com.zaxxer</groupId>
-            <artifactId>HikariCP-java6</artifactId>
-            <version>2.3.9</version>
+            <artifactId>hibernate-jpamodelgen</artifactId>
```

Comment:
I didn't add it. I think it has been added because of the problem with the generate model sources, but I can remove it if it's not necessary.